core/src/main/java/org/apache/calcite/sql/SqlIterationStmt.java

```diff
@@ -40,7 +40,8 @@
    * Creates a {@code SqlIterationStmt}.
    *
    * @param pos         Parser position, must not be null.
-   * @param condition   Conditional expression, must not be null.
+   * @param condition   Conditional expression, can be null for loop statement
```

Comment:
nit: This base class should have no information/context about its subclasses. Suggest simplifying this comment to just "Conditional expression."

Reply:
Make sense. Updated in https://github.com/googleinterns/calcite/pull/287/commits/ee4958b16cdd8fb04f9d6c925b4ccf26767b6cc6